File: benchmark/comparison/tasks/15/solution.py
def fmt_list(lst):
    return "[" + ", ".join(str(x) for x in lst) + "]"

def list_reverse(lst):
    return lst[::-1]

print(fmt_list(list_reverse([1, 2, 3])))
print(fmt_list(list_reverse([])))
print(fmt_list(list_reverse([1])))

File: benchmark/comparison/tasks/15/solution.ts
function formatList(arr: number[]): string {
  return "[" + arr.join(", ") + "]";
}

function list_reverse(arr: number[]): number[] {
  return [...arr].reverse();
}

console.log(formatList(list_reverse([1, 2, 3])));
console.log(formatList(list_reverse([])));
console.log(formatList(list_reverse([1])));
